src/system_info/system_info_network_desktop.cc

```diff
@@ -228,12 +224,10 @@ void SysInfoNetwork::SendUpdate(guint new_device_type) {
   PostMessageToListeners(output);
 }
 
-void SysInfoNetwork::OnNetworkManagerSignal(GDBusProxy* proxy,
-                                            gchar* sender,
-                                            gchar* signal,
-                                            GVariant* parameters,
-                                            gpointer user_data) {
-  SysInfoNetwork* self = reinterpret_cast<SysInfoNetwork*>(user_data);
+void SysInfoNetworkDesktop::OnNetworkManagerSignal(GDBusProxy* proxy,
+    gchar* sender, gchar* signal, GVariant* parameters, gpointer user_data) {
```

Comment:
If you keep the parameters alignment, is the length longer than 80? If no, keep the old format. It's more clear.